launch/handeye_file.launch

```diff
@@ -30,6 +30,8 @@
 
   <!-- The actual transform result you are solving for is saved with this filename -->
   <arg name="calibrated_filename"          default="CalibratedTransform.yml" />
+  <!-- tag the solver summary along the transformm in the calibrated filename -->
```

Comment:
Second question, can calibrated outputs still be re-loaded easily when this is true? I'm sure it is pretty easy to manually update anyway, but we should note that limitation if it exists.